easybuild/easyconfigs/o/OpenColorIO/OpenColorIO-1.1.0-foss-2018b.eb

```
@@ -0,0 +1,29 @@
+easyblock = 'CMakeMake'
+
+name = 'OpenColorIO'
+version = '1.1.0'
+
+homepage = 'http://opencolorio.org/'
+description = """ OpenColorIO (OCIO) is a complete color management solution geared towards motion picture production
+ with an emphasis on visual effects and computer animation."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+source_urls = ['https://github.com/imageworks/OpenColorIO/archive/']
+sources = ['v%(version)s.tar.gz']
+patches = ['OpenColorIO-1.1.0_error.patch']
```

@smoors Please use `%(version)s` when possible:

```suggestion
patches = ['OpenColorIO-%(version)s_error.patch']